### PR TITLE
cleanup

### PR DESCRIPTION
**Pull request by SimonCropp**

### CODE REVIEW

File: src/Transport/NServiceBus.AzureStorageQueues.csproj [hunk 1]
```diff
@@ -20,10 +17,7 @@
 
   <ItemGroup>
     <!-- lock WindowsAzure.Storage to version 8.0.0 -->
-    <PackageReference Include="WindowsAzure.Storage" Version="8.0.0" />
-    <PackageReference Include="Microsoft.Azure.KeyVault.Core" Version="2.0.4" />
-    <PackageReference Include="Microsoft.Data.Services.Client" Version="5.8.2" />
-    <PackageReference Include="System.Linq.Queryable" Version="4.3.0" />
+    <PackageReference Include="WindowsAzure.Storage" Version="[8.4.0, 9.0.0]" />
```
Comment (SeanFeldman) [inline src/Transport/NServiceBus.AzureStorageQueues.csproj]: Not good. We want to lock it to 8.0.0

File: src/AcceptanceTests/NServiceBus.AzureStorageQueues.AcceptanceTests.csproj [hunk 2]
```diff
@@ -16,7 +14,7 @@
     <PackageReference Include="NUnit3TestAdapter" Version="3.8.0-alpha1" />
 
     <!-- Force tests to run with the latest Storage library -->
-    <PackageReference Include="WindowsAzure.Storage" Version="[8.*, 9.0.0]" />
+    <PackageReference Include="WindowsAzure.Storage" Version="8.*" />
```
Comment (SeanFeldman) [inline src/AcceptanceTests/NServiceBus.AzureStorageQueues.AcceptanceTests.csproj]: This is what I had and you said to use a range 😄

Comment (SimonCropp) [reply src/AcceptanceTests/NServiceBus.AzureStorageQueues.AcceptanceTests.csproj]: I said use a range on the main project

File: src/Transport/NServiceBus.AzureStorageQueues.csproj [hunk 3]
```diff
@@ -20,10 +17,7 @@
 
   <ItemGroup>
     <!-- lock WindowsAzure.Storage to version 8.0.0 -->
-    <PackageReference Include="WindowsAzure.Storage" Version="8.0.0" />
-    <PackageReference Include="Microsoft.Azure.KeyVault.Core" Version="2.0.4" />
-    <PackageReference Include="Microsoft.Data.Services.Client" Version="5.8.2" />
-    <PackageReference Include="System.Linq.Queryable" Version="4.3.0" />
```
Comment (SeanFeldman) [inline src/Transport/NServiceBus.AzureStorageQueues.csproj]: Removing these 2 causes [failure](https://builds.particular.net/viewLog.html?buildId=229964&tab=buildResultsDiv&buildTypeId=NServiceBus_Transports_NServiceBusAzureStorageQueues_1Build) on the build server.

**Pull request closed without merging**